init.lua

```diff
@@ -177,53 +177,64 @@ function ranks.update_nametag(player)
 	end
 end
 
--- [function] Set player rank
+-- [function] Set player rank TODO
 function ranks.set_rank(player, rank)
+	local name = player
```

Comment:
Instead of creating a new variable, you may as well just rename `player` to `name` :man_shrugging:

Reply:
It's overwritten on line 185 I believe.

Reply:
Yeah, you should be able to safely change all instances in which `player` is used to `name` though, including on line 185.

Reply:
Will do!